### PR TITLE
custom  java per instance

### PR DESCRIPTION
**Pull request by Ladvace**

### CODE REVIEW

File: src/common/modals/InstanceManager/Overview.js [hunk 1]
```diff
@@ -175,6 +186,10 @@ const Overview = ({ instanceName, background, manifest }) => {
   const [javaLocalArguments, setJavaLocalArguments] = useState(
     config?.javaArgs
   );
+  const [customJavaPathSwitch, setCustomJavaPathSwitch] = useState(
+    config?.customJavaPath !== undefined
```
Comment (blarfoon) [inline src/common/modals/InstanceManager/Overview.js]: why not `Boolean(config?.customJavaPath)`?

Comment (blarfoon) [reply src/common/modals/InstanceManager/Overview.js]: Also what's the point on having this variable when you can just check `Boolean(config?.customJavaPath)` of `customJavaPath`?

File: src/common/modals/InstanceManager/Overview.js [hunk 2]
```diff
@@ -501,9 +530,56 @@ const Overview = ({ instanceName, background, manifest }) => {
...
+                }
+              }}
+            />
+          </JavaManagerRow>
+          {customJavaPathSwitch && (
+            <JavaManagerRow>
+              <Input
+                value={customJavaPath}
+                onChange={e => {
+                  setCustomJavaPath(e.target.value);
```
Comment (blarfoon) [inline src/common/modals/InstanceManager/Overview.js:557]: What if you type the path and don't actually use the button? The changes won't be saved to redux...

**Review by blarfoon — CHANGES_REQUESTED** [review]

File: src/common/modals/InstanceManager/Overview.js (hunk 1 above)
Comment (Ladvace) [reply src/common/modals/InstanceManager/Overview.js]: I did that, in the same way, it's done for JavaMemory

**Pull request merged**